### PR TITLE
Add uwebsockets/18.3.0

### PR DESCRIPTION
**Pull request by theirix**
Specify library name and version:  **uwebsockets/18.3.0**

It is a header-only library. Waiting for usockets #2012 to be ready for test_package.

- [x] I've read the [guidelines](https://github.com/conan-io/conan-center-index/wiki#how-to-submit-a-pull-request) for contributing.
- [x] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [x] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [x] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

**Comment by theirix** [comment]
@uilianries recipe is unblocked now after merging usockets. Could you please remove the label?

File: recipes/uwebsockets/all/conanfile.py [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+            raise ConanInvalidConfiguration(
+                "%s requires a compiler that supports at least C++%s" % (self.name, minimal_cpp_standard))
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        os.rename("uWebSockets-%s" % self.version, self._source_subfolder)
+
+    def package(self):
+        self.copy(pattern="LICENSE", dst="licenses", src=self._source_subfolder)
+        self.copy(pattern="*.h", src=os.path.join(self._source_subfolder, "src"), dst="include/uWebSockets", keep_path=False)
```
Comment (uilianries) [inline recipes/uwebsockets/all/conanfile.py:55]:
```suggestion
        self.copy(pattern="*.h", src=os.path.join(self._source_subfolder, "src"), dst=os.path.join("include", "uWebSockets"), keep_path=False)
```

File: recipes/uwebsockets/all/conanfile.py [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+        self.copy(pattern="*.hpp", src=os.path.join(self._source_subfolder, "src", "f2"), dst="include/uWebSockets/f2", keep_path=False)
```
Comment (uilianries) [inline recipes/uwebsockets/all/conanfile.py:56]:
```suggestion
        self.copy(pattern="*.hpp", src=os.path.join(self._source_subfolder, "src", "f2"), dst=os.path.join("include", "uWebSockets", "f2"), keep_path=False)
```

File: recipes/uwebsockets/all/test_package/test_package.cpp [hunk 3]
```diff
@@ -0,0 +1,7 @@
+#include "libusockets.h"
+#include "App.h"
```
Comment (uilianries) [inline recipes/uwebsockets/all/test_package/test_package.cpp:2]: terrible, but it's the upstream idea ...

Comment (theirix) [reply recipes/uwebsockets/all/test_package/test_package.cpp:1]: uwebsockets examples usually require this include but ours don't. Dropped the include.

File: recipes/uwebsockets/all/conanfile.py (hunk 1 above)
Comment (theirix) [reply recipes/uwebsockets/all/conanfile.py]: Yes, silly mistake, thanks!

**Comment by uilianries** [comment]
@theirix Any PR requires 3 approvals.

**Comment by theirix** [comment]
@uilianries I think there was a minor UI problem after accepting two suggests in a second. Now it is okay.

Thanks for accepting!

**Pull request merged**